content/articles/getting-started-with-kali-linux/index.md

```diff
@@ -0,0 +1,290 @@
+### Getting started with Kali Linux
+
+**Kali Linux** is the most popular penetration testing platform that provides a variety of security auditing tools used by security specialists and hackers in day-to-day encounters.
+
+Tools available on Kali Linux enable the user to gather information, perform exploits and prevent their devices from exploits.
+
+### Installing Kali Linux
+
+For beginners, it is advised to have your Kali Linux in a **virtual box**. 
+You can have your Kali Linux in your favorite virtual box. Available virtual boxes include:
+
+  * Vmware virtual box
+  * Hyper V virtual box
+  * Kernel-based virtual box 
+  
+For our tutorial, we will use VMware virtual box (My favorite).
+
+#### Download Kali Linux iso
+
+First, before installing the VMware you have to download the Kali Linux iso. Use the link below:
+
+  * Kali Linux iso.
+
+#### Download Vmware virtual box
+After successfully downloading the kali Linux iso, you can now download the VMware virtual box using the link below.
+
+  * Vmware virtual box.
+
+Follow the "Installing Vmware " tutorials to successfully install the VMware virtual box.
+
+#### Installing Kali Linux in Vmware virtual box
+
+You now have all the requirements to successfully set up Kali Linux inside the VMware virtual box.
+
+Lastly, follow the guidelines in the "Installing Kali Linux in VMware " to successfully install Kali Linux.
+
+![Login page](/engineering-education/getting-started-with-kali-linux/login.png)
+
+And Cheers! You have successfully installed Kali Linux and now ready to learn.
+
+### Starting Kali Linux
+
+On your virtual box, select the Kali Linux option and click the play button to start your Kali Linux.
+Enter your username and password when prompted.
+
+**Note:** The **default** username is **root** and the default password is **toor**.
+
+#### Updating Kali Linux
+
+It is recommended to frequently update your Kali Linux and its tools to the latest versions to keep it up to date. 
+Follow the steps below to update and upgrade Kali Linux:
+
+  * Start the **terminal** by:
+        
+       * Clicking the **cltr+alt+T** buttons simultaneously or
+       * Using the GUI(Graphical User Interface), go to the applications section and double click on the terminal option.
+
```

Comment:
This is misleading. Are we really clicking cltr+alt+T ? Press or hold is preferred in this case.

Reply:
@Kimosop-Wakanda  You haven't edited this bit